jvm/src/main/scala/cats/jvm/EvalAsync.scala

```diff
@@ -0,0 +1,41 @@
+package cats
+package jvm
+
+import data.Reader
+import java.util.concurrent.{Callable, CountDownLatch, ExecutorService}
+
+object EvalAsync {
+  def apply[A](cb: (A => Unit) => Unit): Eval[A] = Eval.always {
+    val cdl = new CountDownLatch(1)
+    var result: Option[A] = None
+    cb((a: A) => {result = Some(a); cdl.countDown})
+    cdl.await
+    result.get // YOLO
+  }
+
+  implicit class EvalFork[A](val eval: Eval[A]) extends AnyVal {
+    def callable(cb: A => Unit): Callable[Unit] = new Callable[Unit] {
```

Comment:
At a glance I have no idea why you're using `Callable` instead of `Runnable` here. Do you actually need the ability to throw checked exceptions?